lib/facter/docker.rb

```diff
@@ -138,7 +138,11 @@ def interfaces
         docker_network_names.each do |network|
           inspect = JSON.parse(Facter::Core::Execution.execute("#{docker_command} network inspect #{network}", timeout: 90))
           docker['network'][network] = inspect[0]
-          network_id = docker['network'][network]['Id'][0..11]
+          if docker['network'][network]['Id'].nil?
+            network_id = docker['network'][network]['id'][0..11]
+       	  else
+            network_id = docker['network'][network]['Id'][0..11]
+       	  end
```

Comment:
I think something like this will satisfy the acceptance test.

```suggestion
          network_id = if docker['network'][network]['Id'].nil?
                         docker['network'][network]['id'][0..11]
                       else
                         docker['network'][network]['Id'][0..11]
                       end
```